sencha-workspace/packages/slate-cbl/src/field/TaskSelector.js

```diff
@@ -20,7 +20,7 @@ Ext.define('Slate.cbl.field.TaskSelector', {
 
 
     store: {
-        type: 'slate-cbl-tasks',
```

Comment:
Taking a look at what configs the store come with that we'd lose by moving to defining a generic store from scratch:

- `remoteFilter: true`
- `pageSize: 0`

Take a look if we need either of these replicated inline here. We probably don't want unlimited pages for the combo...not sure if remoteFilter is needed or desirable, are we relying on local filtering here or sending the query to the server?

Reply:
@themightychris the store definition in the task selector already had `remoteFilter: true` specified.   I added the `pageSize: 0` config because there is no paging toolbar mechanism in this task selector select box and I assume we want to return all results.   Let me know if you see it differently.